titus-server-master/src/main/java/com/netflix/titus/master/mesos/MesosConfiguration.java

```diff
@@ -92,4 +86,10 @@
      */
     @DefaultValue("./run")
     String getExecutorUriOverrideCommand();
+
+    /**
+     * @return the registry URL that will be merged with the image name.
+     */
+    @DefaultValue("")
+    String getRegistryUrl();
```

Comment:
Is this expected to also include the port suffix on the URL?

Reply:
yes, it would be the entire string that is prepended to the image name